config.go

```diff
@@ -14,12 +14,13 @@ const ConfigFile = "faas.yaml"
 // Config represents the serialized state of a Function's metadata.
 // See the Function struct for attribute documentation.
 type config struct {
-	Name      string `yaml:"name"`
-	Namespace string `yaml:"namespace"`
-	Runtime   string `yaml:"runtime"`
-	Image     string `yaml:"image"`
-	Trigger   string `yaml:"trigger"`
-	Builder   string `yaml:"builder"`
```

Comment:
Pretty sure it was my IDE that formatted these like this. What IDE are you using and do you have a Golang extension/plugin?

Reply:
I am using Goland (Intellij based IDE), but I called `go fmt` from it. Previous formatting was also correct. The indentation changed because I added new member
```go
BuilderMap map[string]string `yaml:"builderMap"`
```

Reply:
Oh of course.